App.js

```diff
@@ -1,36 +1,26 @@
-import * as React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
+import React from 'react'
+import { createAppContainer } from 'react-navigation';
+import { createStackNavigator } from 'react-navigation-stack';
+import HomeScreen from './src/screens/HomeScreen';
+import UserRegScreen from './src/screens/UserRegScreen'
+import OwnerRegScreen from './src/screens/OwnerRegScreen';
+const navigator = createStackNavigator(
+  {
+    Home: HomeScreen,
```

Comment:
We will probably want to change "HomeScreen" to "WelcomeScreen" or whatever, because once a user is logged in the home screen won't have the sign up prompt (it will be the map view?)